packages/deploy/deploy/800_oft_sand/802_setup_sepolia.ts

```diff
@@ -0,0 +1,35 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {ethers} from 'ethers';
+import OFTSandBaseJson from '../../deployments/baseSepolia/OFTSand.json';
+import OFTSandBscJson from '../../deployments/bscTestnet/OFTSand.json';
+
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute} = deployments;
+  const {deployer} = await getNamedAccounts();
+
+  const eidBaseSepolia = 40245;
+  const eidBscTestnet = 40102;
+
+  await execute(
+    'OFTAdapterForSand',
+    {from: deployer, log: true},
+    'setPeer',
+    eidBaseSepolia,
+    ethers.zeroPadValue(OFTSandBaseJson.address, 32)
+  );
+
+  await execute(
+    'OFTAdapterForSand',
+    {from: deployer, log: true},
+    'setPeer',
+    eidBscTestnet,
+    ethers.zeroPadValue(OFTSandBscJson.address, 32)
+  );
+};
+
+export default func;
+func.tags = ['OFT_Sepolia_setup'];
```

Comment:
Change to: `func.tags = ['OFTAdapterForSand', 'OFTAdapterForSand_sepolia_setup'];`
Add: `func.dependencies = ['OFTAdapterForSand_deploy'];`

Reply:
One file per network for now (change tags as needed)

Reply:
Rename file to `802_oft_adapter_setup_baseSepolia` (or for whichever network this file is intended for) and update the other deploy file names as well so it's obvious which contract/network is being setup